BrowserKit/Sources/ToolbarKit/ToolbarButton.swift

```diff
@@ -162,11 +189,24 @@ class ToolbarButton: UIButton, ThemeApplicable {
         return element.isFlippedForRTL ? image?.imageFlippedForRightToLeftLayoutDirection() : image
     }
 
-    private func removeAllGestureRecognizers() {
-        guard let gestureRecognizers else { return }
-            for recognizer in gestureRecognizers {
-                removeGestureRecognizer(recognizer)
-            }
+    private func removeLongPressGestureRecognizer() {
+        guard let recognizer = longPressRecognizer, let interaction = largeContentViewerInteraction else { return }
+        removeGestureRecognizer(recognizer)
+        longPressRecognizer = nil
+
+        removeInteraction(interaction)
+        largeContentViewerInteraction = nil
+    }
+
+    private func setMinimumPressDuration() {
+        // The default long press duration is 0.5. Here we extend it if
+        // UILargeContentViewInteraction is enabled to allow the large content
+        // viewer time to display the content
+        var minimumPressDuration: TimeInterval = UX.defaultMinimumPressDuration
+        if UILargeContentViewerInteraction.isEnabled {
+            minimumPressDuration = UX.minimumPressDurationWithLargeContentViewer
+        }
+        longPressRecognizer?.minimumPressDuration = minimumPressDuration
```

Comment:
I believe we can make good use of the ternary operator here:
`        longPressRecognizer?.minimumPressDuration = UILargeContentViewerInteraction.isEnabled ? UX.minimumPressDurationWithLargeContentViewer : UX.defaultMinimumPressDuration
`

Reply:
That's what I tried first but the line was way too long and wrapping it looked weird.